INSTALL.md

```diff
@@ -52,6 +52,8 @@ NEW or CHANGED MINIMUM dependencies since the last major release are **bold**.
  * If you want support for HEIF/HEIC images:
      * libheif >= 1.3 (tested through 1.9; older versions may also work, we
        haven't tested)
+ * If you want support for AVIF images:
+     * libheif >= 1.7 (tested through 1.10)
```

Comment:
Can you combine?

     * If you want support for HEIF/HEIC or AVIF images:
         * libheif >= 1.3 (1.7 required for AVIF support, tested through 1.10)